src/index.js

```diff
@@ -76,10 +76,26 @@ const getEntity = (entityOrId, schemaKey, entities, isImmutable) => {
     entities[schemaKey][entityOrId];
 };
 
-const getEntities = (entities, isImmutable) => (schema, entityOrId) => {
-  const schemaKey = schema.key;
-
-  return getEntity(entityOrId, schemaKey, entities, isImmutable);
+const getEntities = (entities, cache, isImmutable) => (schema, entityOrId) => {
+  return (handleCacheHit, handleCacheMiss) => {
+    const schemaKey = schema.key;
+    const getEntityId = (entity) => typeof entity === 'object' ? schema.getId(entity) : entity;
```

Comment:
It looks like @paularmstrong handled this in the branch that he was trying to implement circular immutable references, which I'll merge into this branch shortly once we've settled on a solution for that.